src/main/java/org/kpmp/geneExpressionSummary/SNRNAParticipantValue.java

```diff
@@ -0,0 +1,37 @@
+package org.kpmp.geneExpressionSummary;
```

Comment:
We should have a simple test on this object as well. There is debate on whether it is worth testing POJOs (Plain Old Java Objects), but I like having them to verify that I didn't mess up what value is being returned/set. Also, it is nice to have them in place if/when additional logic may be introduced into the object.